docs/Contributing/pull-request.rst

```diff
@@ -149,14 +154,22 @@ a change to another file that you didn't intend to actually change. If you find
 a file, you can use the :file:`git restore <file>` command. This is also shown on the :file:`git status`
 command output.
 
-Once everything looks good from a :file:`git status` perspective, then issue the last two commands. The last
-command, :file:`git commit` will open a text editor (in Debian based systems this is often Nano). Put a message
-at the bottom below all of the hashes ( # ) and then save.
+The :file:`git add --all` command then adds all the changed files into a single commit. The
+:file:`git clang-format` command then runs a pre-processor to format any code changes to support the
+Longturn community's code standards. We use ``clangformat-11`` by default, if your system's version of
+``clang-format`` is different, you can use the binaries for ``clangformat-11`` using the command:
+:file:`git clang-format --binary <path-to-clang-format-11>`. If :file:`git clang-format` modifies any files,
+it will tell you so. If that happens, then you need to run a subsequent :file:`git add --all` command to get
+those files added back into the commit.
+
+Once everything looks good from a :file:`git status` perspective, then issue the last two command. The last
+command, :file:`git commit`, will open a text editor (in Debian based systems this is often ``Nano``). Put a
+message at the bottom below all of the hashes ( ``#`` ) and then save.
 
-At this point you have a commit of changes that you need to push to Origin.
+At this point you now have a commit of changes that you need to push to Origin.
```

Comment:
I'd suggest either
```suggestion
You now have a commit of changes that you need to push to Origin.
```
or to leave it as it was before. "At this point you now have" sounds weird, since "at this point you have" and "you now have" mean effectively the same thing.